content/en/docs/virtualization.md

```
@@ -334,3 +334,121 @@ windows.2k22 | Microsoft Windows Server 2022
+```
+ovn.kubernetes.io/routes: |
+  [{
+    "dst": "10.10.13.0/24",
+    "gw": "10.244.3.12"
+  }]
+```
+
+Custom routes will be applied to all pods in the tenant namespace.
+
+
+## Virtual Routers
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)